packages/odyssey-react-mui/src/Buttons/BaseMenuButton.tsx

```diff
@@ -174,31 +215,52 @@ const MenuButton = ({
         id={`${uniqueId}-button`}
         isDisabled={isDisabled}
         label={buttonLabel}
+        children={buttonChildren}
```

Comment:
**NON-BLOCKING**
I'd like to fix the types for if `children` or `label`, but not in this PR. Bryan's working on this already.

My comment is here for historical purposes.